ccmbioinfo/lfs
# CARDS-1904: Reference questions: add a listener that captures the changes to the source answer and updates the copied answer

[CARDS-1904](https://phenotips.atlassian.net/browse/CARDS-1904)

To test:

- in test mode create Reference Test - User Answers and Reference Test - Copied Answers forms
- change any answer of the Reference Test - User Answers form and check if the appropriate answer in Reference Test - Copied Answers form was changed as well 

- in prems mode, create a new Survey Events form -> missing answers for Hospital, Department and Assigned survey
- create a new Visit Information form, fill in the values and save -> the Survey Events form gets updated with the new values
- edit the VI form, change the values (use different multiple values for Provider) -> SE also gets updated
- edit the VI form, remove the values -> SE values also get removed

[CARDS-1904]: https://phenotips.atlassian.net/browse/CARDS-1904?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
